### DISCUSSION TITLE
Improve installation documentation about GestureHandlerRootView

### DISCUSSION THREAD

**Discussion by francois-pasquier** [software-mansion/react-native-gesture-handler#1947] [Ideas]
Maybe I was distracted when I followed the installation instructions but I think this example should include the styling `flex: 1`

```tsx
export default function App() {
  return <GestureHandlerRootView>{/* content */}</GestureHandlerRootView>;
}
```

Putting it as the end of the `INFO` section makes it easy to miss imo : )

**Comment by SMhdAsadi** [comment]
It is not required to put flex:1 for any GestureHandlerRootView. I think this is why they didn't put it here.

**Reply by hirbod** [reply]
Not aware of any app that would work without `flex: 1` (or flexGrow: 1)
